src/CiviEntityStorage.php

```diff
@@ -74,14 +78,15 @@ public static function createInstance(ContainerInterface $container, EntityTypeI
     return new static(
       $entity_type,
       $container->get('database'),
-      $container->get('entity.manager'),
+      $container->get('entity_field.manager'),
```

Comment:
Why doesn't this explode if the construct call above still type hints `EntityManagerInterface`?